### DISCUSSION TITLE
lolMiner 1.69

### DISCUSSION THREAD

**Discussion by Lolliedieb** [Lolliedieb/lolMiner-releases#1879] [General]
### Changes

- Added Nexa support for AMD Vega  / VII (1) and RDNA 1-3 GPUs
- The parameter **--keepfree** can now be used to keep a certain amount of memory free on each GPU on Nexa mining. This might end up in allocating a smaller lookup table, so the GPU memory has space left for other workloads (2)
- The --keepfree parameter is now working for each GPU separately. Use a comma separated list of values if you want to assign different values for each GPU.
- Nexa mining will now allocate memory for each GPU sequentially and delay later cards startup slightly to help systems with small virtual memory.

### Notes
(1) AMD Vega and VII are currently only supported on GPU drivers running the Rocm compute OpenCL driver stack. This is e.g. default for Vega GPUs on the latest Linux drivers, but not on pre 2021 drivers. Support for older drivers (and maybe older GPUs) might be included with the upcoming releases. 
(2) For users of the alternative Windows version of 1.68: Use --keepfree 1024 or higher to get the effect the alternative version had.

General note: The Nexa gpu codes are (and will be in foreseeable future) joint work with Iedoc from [BzMiner](https://github.com/bzminer/bzminer). Miner fees will be evenly shared regardless of which of these two flavors of the code you prefer.

<hr /><em>This discussion was created from the release <a href='https://github.com/Lolliedieb/lolMiner-releases/releases/tag/1.69'>lolMiner 1.69</a>.</em>

**Comment by OrionUA** [comment]
Nexa + zil windows?

**Comment by bergenion** [comment]
how config NEXA+ZIL on HiveOS om RDNA2 ? Miner losts connection to nexa pool alltime

**Comment by Olegly** [comment]
Trojan:Win32/Wacatac.A!ml
lolMiner_v1.69_Win64.zip

**Reply by tanerileri69** [reply]
Can you show an example for keepfree?
Let's say there are 4 gpu, 6700xt 6700xt 6900 xt 6700 xt.
should i use keepfree? if yes how?

**Comment by Lolliedieb** [comment]
As with every version before .... antivirus soft does not like mining
software, especially not when it is packed...

Am Sa., 4. März 2023 um 10:17 Uhr schrieb Olegly ***@***.***>:

**Comment by noebranco** [comment]
Adding Dynex soon?

**Comment by tanerileri69** [comment]
Can you show an example for keepfree?
Let's say there are 4 gpu, 6700xt 6700xt 6900 xt 6700 xt.
should i use keepfree? if yes how?
